system-tests/test.py

```diff
@@ -206,7 +206,7 @@ def __init__(self,
                 configfile.write('ld-flags = {}\n'.format(ld_flags))
             else:
                 configfile.write('ld-flags = ' + '\n')
-
+        self.conf_file = conf_file
```

Comment:
Could replace `conf_file` with `self.conf_file` the whole way through?  Not a major issue but it would save having an extra intermediate variable